src/main/scala/firrtl/Emitter.scala

```diff
@@ -92,6 +92,14 @@ class VerilogEmitter extends Emitter {
     }
     case _ => error("Shouldn't be here")
```

Comment:
I didn't touch that line, I was kind of leaving it for whenever the Emitter gets ultimately reworked